cms_form/tests/utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2017-2018 Simone Orsi
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+
+from io import StringIO
+from werkzeug.wrappers import Request
+import mock
+import urllib.parse
```

Comment:
still called urlparse in 2.7

Reply:
I've already fixed all this stuff but in the last commit here #62 and I missed it in the split :sob: